fake_dependencies/selfservice-api/src/routes/capabilities.ts

```diff
@@ -6,6 +6,14 @@ import { composeUrl, createId, getDate, log } from "../helpers";
 const router = express.Router();
 
 router.get("/capabilities", (req, res) => {
+  // res
+  //   .set("Content-Type", "application/problem+json")
+  //   .status(400)
+  //   .send({
+  //     title: "title of error",
+  //     detail: "this is the detail of the error"
+  //   });
+
```

Comment:
Can you add a boolean toggle like you did in kafkatopics.ts to make it easier to test error handling functionality?